services/oada_upload/tsconfig.json

```diff
@@ -14,6 +14,6 @@
     "forceConsistentCasingInFileNames": true,
     "allowSyntheticDefaultImports": true
   },
-  "include": ["src/**/*"],
+  "imclude": [ "src/**/*" ],
```

Comment:
typo? I'm not sure the `include` line is even though -- I believe the default is fine.

Reply:
you are right, it is a typo